scripts/xapi.service

```diff
@@ -1,6 +1,21 @@
 [Unit]
 Description=XenAPI server (XAPI)
-After=syslog.target forkexecd.service xcp-networkd.service message-switch.service v6d.service save-boot-info.service attach-static-vdis.service genptoken.service xcp-rrdd.service xapi-storage-script.service xenopsd-xc.service xenstored.service xcp-networkd.service
```

Comment:
The manual says:

```
This option may be specified more than once, in which case ordering dependencies for
all listed names are created.
```

My understanding is: all After constraints must be met but their ordering is not important. Hence, xapi starts after all of them are reached.